gatsby-node.js

```diff
@@ -44,14 +47,17 @@ const createHomePage = async (createPage) => {
 // TODO: add action for creating article pages
 const createArticles = async (createPage) => {
   const newsResp = await axios.get(`${BASE_URL}https://www.thedp.com/section/news.json`)
+  const mostReadDPResp = await axios.get(`${BASE_URL}https://us-central1-web-services-dp.cloudfunctions.net/dropcap/DP`)
+
   newsResp.data.articles.forEach(article => {
     const { authors } = article
     authors.forEach(async author => await createAutors(createPage, author.slug))
```

Comment:
Same thing I said to @nitrateatom, Promise.all seems to make more sense here.

Reply:
i wrote this code, i personally prefer using `async await` instead of `Promise.all`. Is there a reason why `Promise.all` is better? If it is, we can change